pages/api/company/[id].js

```diff
@@ -13,6 +13,7 @@ const apiSchema = Joi.object({
 });
 
 const updateCompany = async (req, res) => {
+  console.log("Update Company Request Start");
```

Comment:
```suggestion
  console.log("updateCompany() Request Start");
```
please use method name in log, so that while debugging we can find this method easily in whole code